src/test/java/SetTest.java

```diff
@@ -0,0 +1,45 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void size() {
+        int result = numbers.size();
+
+        assertThat(result).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "2", "3"})
+    void contains(int number) {
+        boolean result = numbers.contains(number);
+
+        assertThat(result).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+    void contains(int number, boolean expected) {
+        boolean result = numbers.contains(number);
+
+        assertThat(result).isEqualTo(expected);
```

Comment:
```suggestion
        assertThat(numbers.contains(number)).isEqualTo(expected);
```

로컬 변수 없이 위와 같이 구현해도 되지 않을까?